Gordon360/ApiControllers/HousingController.cs

```diff
@@ -324,7 +324,7 @@ public IHttpActionResult GetApartmentApplication(int applicationID)
         /// <returns>Object of type ApartmentApplicationViewModel</returns>
         [HttpGet]
         [Route("admin/apartment/applications/")]
-        [StateYourBusiness(operation = Operation.READ_ALL, resource = Resource.HOUSING)]
+        [StateYourBusiness(operation = Operation.READ_ALL, resource = Resource.HOUSING_ADMIN)]
```

Comment:
I think this is incorrect. This method is to read all apartment applications, so it's accessing the `HOUSING` resource, not the `HOUSING_ADMIN` resource. Perhaps this got confused because only someone who is a housing admin can access this?

Reply:
Ah... that makes sense. Yeah, that was my confusion.